src/main/java/jdbc/pojo/Users.java

```diff
@@ -0,0 +1,74 @@
+package jdbc.pojo;
+
+import java.sql.Timestamp;
+import java.time.LocalDate;
+import java.util.Objects;
+
+public class Users  {
```

Comment:
why users? not user?

Reply:
I didn't find annotation for table and decided it only in hibernate/ That's why i had to call it same as table name

Reply:
try to rename just class name to User.